samples/estimation/df-chemistry/README.md

```diff
@@ -0,0 +1,16 @@
+# Resource Estimation for Double-factorized Chemistry
+
+In this notebook we evaluate the physical resource estimates of using the so-called double-factorized qubitization algorithm described in [[Phys. Rev. Research 3, 033055 (2021)](https://doi.org/10.1103/PhysRevResearch.3.033055)] to calculate the energy of a user provided Hamiltonian to chemical accuracy of 1 mHa. 
+
+The Hamiltonian is provided as an FCIDUMP file that is available on your machine or can be downloaded via an HTTPS URL.
```

Comment:
Could we give a usage example on how to call the script with a filename or an URI?